M2/CMakeLists.txt

```diff
@@ -58,7 +58,7 @@ file(STRINGS VERSION VERSION)
 project(Macaulay2
   VERSION	${VERSION}
   DESCRIPTION	"A software system for algebraic geometry research"
-  HOMEPAGE_URL	http://macaulay2.com # contact: Macaulay2@math.uiuc.edu
+  HOMEPAGE_URL	https://macaulay2.com # contact: Macaulay2@math.uiuc.edu
```

Comment:
The email also needs to change, or just be removed, but not very urgent.

Reply:
Easy enough -- I added another commit fixing it, plus one other location I found with this old address.